replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+
+### Study 1a
+
+
+> In this study, there were 103 participants (mean age = 29.3 years; median household size = 3 people; median household income = $35,000). One participant was excluded as an outlier on income (more than 3 SD from the mean). We conducted binary logistic regressions to compare how frequently participants cited each of the two considerations of interest (trade-offs vs. location) as a function of income.
+
+```{r echo = FALSE}
+# Data import
+
+# read in the data for 1a using the spss file
+# the warning/messages suppressed; the data imports cleanly.
+suppressMessages(suppressWarnings(raw_1a_spss <- read.spss("framing/study_1a.sav", to.data.frame = TRUE)))
+suppressMessages(suppressWarnings(raw_1a_csv <- read.csv("study_1a_demog.csv", header = FALSE)))
+
+#calculate mean age
+raw_1a_csv %>%
+  summarise(average_age = mean(V3)) %>%
+  round(digits = 1)
+
+#calculate sex breakdown
+raw_1a_csv %>%
+  group_by(V4) %>%
+  summarise(sex_breakdown = n())
+
+#calculate sample size, median household income, median household size
+raw_1a_spss %>%
+  summarise(sample.size = n(), median.household.income = median(income2), median.household.size = median(house))
+
+clean_1a <- raw_1a_spss %>%
+  filter(filter_. != "Not Selected")
+```
+
+> Higher-income participants were more likely than lower-income participants to name location as the main consideration, β = 2.52, Wald-test χ2(1, N = 102) = 5.54, p < .05,
+
+```{r echo = FALSE}
+glm(location ~ logses, data = clean_1a, family = "binomial") %>%
+  tidy() %>%
+  filter(term == "logses") %>%
+  mutate(beta = round(estimate, digit = 2),
+         p = round(p.value, digit = 4)) %>%
+  select(term, beta, p)
+
+logit1logses <- glm(location ~ logses, data = clean_1a, family = "binomial")
+
+print(wald.test(b = coef(logit1logses), 
+                Sigma = vcov(logit1logses), 
+                Terms = 2)$result$chi2[1], digit = 3)
+```
+
+> but lower-income participants were more likely than higher- income participants to name trade-offs as the main consideration, β = −1.62, Wald-test χ2(1, N = 102) = 4.67, p <.05.
+
+```{r echo = FALSE}
+glm(tradeoff ~ logses, data = clean_1a, family = "binomial") %>%
+  tidy() %>%
+  filter(term == "logses") %>%
+  mutate(beta = round(estimate, digit = 2),
+         p = round(p.value, digit = 4)) %>%
+  select(term, beta, p)
+
+logit2logses <- glm(tradeoff ~ logses, data = clean_1a, family = "binomial")
+
+print(wald.test(b = coef(logit2logses), Sigma = vcov(logit2logses), Terms = 2)$result$chi2[1], digit = 3)
+```
+
+```{r echo = FALSE}
+figure1alow <- clean_1a %>%
+  filter(income2 <= 35000) %>%
+  mutate(total.n = n(), total.tradeoff = sum(tradeoff), prop.tradeoff = sum(tradeoff)/n(), total.location = sum(location), prop.location = sum(location)/n(), SE.tradeoff = sqrt(prop.tradeoff * (1 - prop.tradeoff)/total.n), CI.tradeoff = qnorm(.975) * SE.tradeoff, SE.location = sqrt(prop.location * (1 - prop.location)/total.n), CI.location = qnorm(.975) * SE.location) %>%
```

Comment:
Consider separating these equations line-by-line so that they are easier to read (and amend).